api/app/controllers/review.controller.ts

```diff
@@ -16,6 +16,29 @@ import {
   buildIsInActiveCondition,
   buildIsPendingCondition,
 } from "../utils/review.inference";
+import { parseMdToJSON } from "../helpers/transcript";
+import axios from "axios";
+import { BaseParsedMdContent, TranscriptAttributes } from "../types/transcript";
+
+
+// THis function fetches and parses a transcript from a URL (already saved in the db which points to transcript on github), or returns the original transcript if no URL is provided. This is use to sync a transcript in review with the FE.
+const transcriptWrapper = async (transcript: TranscriptAttributes) => {
+  if (!transcript.transcriptUrl) return transcript;
```

Comment:
This method now gives two different types of responses, which isn't right.

When there's no `transcriptURL`, the response stays the same as before. But now, if `transcriptURL` is there, the response changes. It won't have the old structure with transcript `{id: ..., content: {}, originalContent: {}` anymore

Reply:
I just made a commit to address this.